src/codeu/chat/client/core/InterestsContext.java

```diff
@@ -0,0 +1,27 @@
+package codeu.chat.client.core;
+
+import java.util.Collection;
+
+import codeu.chat.common.BasicController;
+import codeu.chat.common.BasicView;
+import codeu.chat.common.ConversationHeader;
+import codeu.chat.common.Interests;
+import codeu.chat.util.Uuid;
+
+
+public final class InterestsContext {
+    public final Interests interests;
+    private final BasicView view;
+    private final BasicController controller;
+
+    public InterestsContext(Interests interests, BasicView view, BasicController controller) {
+        this.interests = interests;
+        this.view = view;
+        this.controller = controller;
+    }
+    public InterestsContext (Interests interests){
```

Comment:
Tired of writing the same words already, so here's a picture: https://upload.wikimedia.org/wikipedia/commons/2/2c/Key_delete.jpg

@lysanderp41